pkg/kubelet/dockershim/network/cni/cni_test.go

```diff
@@ -45,6 +46,7 @@ import (
 	networktest "k8s.io/kubernetes/pkg/kubelet/dockershim/network/testing"
 	"k8s.io/utils/exec"
 	fakeexec "k8s.io/utils/exec/testing"
+
```

Comment:
Another blank line that could be removed.

Reply:
Thx